### ISSUE TITLE
intuition behind trainable weights and bias for losses

### ISSUE THREAD

**Issue by DikshaMeghwal** [joonson/syncnet_trainer#25]
Hey
I was wondering if you could shed some light on why did you add learnable weights and biases to the sync and identity losses? To me it seemed like you were possibly trying to scale and shift but I don't understand why the model doesn't train without it. 
Also if you put learnable weights and biases, whats stopping the weight to be 0 and making loss 0?
I am using voxceleb dataset for training.

Below are the loss curves for when I removed the weights. The sync loss seems to be stagnant while the identity loss is increasing.
<img width="718" alt="Screenshot 2024-04-15 at 12 46 35 PM" src="https://github.com/joonson/syncnet_trainer/assets/6906542/b807d275-fdb5-4d3d-a409-6731b4d2136d">
<img width="717" alt="Screenshot 2024-04-15 at 12 46 51 PM" src="https://github.com/joonson/syncnet_trainer/assets/6906542/38438e0d-850e-4334-8e4b-74e209bd7755">

**Comment by 6eternal6** [comment]
hello, 
Seeing that you have successfully trained this code, I would like to ask what is the meaning of offset in dataLoader() when processing the data? Is this taken from the txt file?
Thank you!
